src/Domain/Task/Handler/RescheduleHandler.php

```diff
@@ -25,7 +25,7 @@ public function __invoke(Reschedule $command)
         $task->setBefore($rescheduledBefore);
         $task->unassign();
         $task->setMetadata('rescheduled', true);
-        $task->setFailureReason(null);
+        $task->setIncidented(false);
```

Comment:
I am surprised by this change : 
1. why should we override the previous "incident statute" if the task was rescheduled? task had an incident and it should be kept 
2. also, I guess rescheduling should create an incident as well, so we can keep track of it. I think it will nice but not 100% sure what do you think ? what @Paul-Eraman-CoopCycle  do you think about registering reschedule as incident ?

Reply:
The actual behavior of the `hasIncidents` fields is to say if the task is in a "error"/incident state. Rescheduling a Task remove this state to avoid polluting the dashboard with task that has been reschuduled for another day.
This is not optimal, but QA-Coops seems happy with this behavior for now. Things will change later with a full incident feature.

Reply:
The `hasIncidents` field indicates whether a task is in an error state or has encountered an incident. To prevent the dashboard from being cluttered with tasks that have been rescheduled for a different day, rescheduling a task will reset this field, removing the error/incident state.
This is not optimal, but QA-Coops seems happy with this behavior for now. Things will change later with a full incident feature.

Reply:
Rescheduling tasks are recorded but not categorized as incidents; they are treated as consequences of incidents. For instance, if a customer is not at home, the rider may report a `NO_CUSTOMER` incident. Subsequently, the dispatcher can address this incident by rescheduling the task for a later time.